source/plugin/project/version-numbers.rst

```diff
@@ -3,16 +3,15 @@ Version Numbers
 
 Your plugin needs a unique version number in order to allow humans to find out which version of a plugin is the
 successor of the other. There are plenty of versioning schemes that you could use for your own plugin. In the following
-part we explain a few concepts with their pros and cons.
+part, we explain a few concepts with their pros and cons.
 
-Sponge recommends to use a version identifier that consists of either two or three numeric version parts separated with
-a ``.`` and additional label appended with a hyphens ``-`` such as ``3.1`` or ``1.0.3-API7`` similar to SemVer.
+Sponge recommends using a version identifier that consists of either two or three numeric version parts separated with
+a ``.`` and additional label appended with a hyphen ``-`` such as ``3.1`` or ``1.0.3-API7`` similar to SemVer.
 Start with ``1.x`` if your plugin is ready. Start with ``0.x`` if your plugin is not yet ready or unstable.
 
 .. note::
 
-    While Sponge recommends using the above versioning scheme, plugin authors are free to choose their own. Please keep
-    in mind that your users have to deal with a multitude of versioning schemes, so keep it simple.
+    While Sponge recommends using the above versioning scheme, plugin authors are free to choose their own. Please keep in mind that your users have to deal with a multitude of versioning schemes, so keep it simple.
```

Comment:
Please keep the line length at 120 characters:

https://docs.spongepowered.org/stable/en/contributing/spongedocs.html?highlight=maximum%20length#style-guide